osbrain/tests/test_agent_transport.py

```diff
@@ -46,9 +48,10 @@ def test_agent_bind_transport_agent(nsproxy):
     address = agent.bind('PUSH')
     assert address.transport == 'tcp'
 
-    agent = run_agent('a1', transport='ipc')
-    address = agent.bind('PUSH')
-    assert address.transport == 'ipc'
+    if os.name == 'posix':
+        agent = run_agent('a1', transport='ipc')
+        address = agent.bind('PUSH')
+        assert address.transport == 'ipc'
```

Comment:
Just use `inproc` instead of `ipc` to make it work in all platforms.

Reply:
Oh. I was convinced 'IPC' and 'inproc' were the same thing. Good thing you pointed it out.

Reply:
`inproc` is for in-process communication (i.e.: between threads). Happens in memory, so it is much faster than IPC.